src/Umbraco.Web.UI.Client/src/common/resources/auth.resource.js

```diff
@@ -34,6 +70,26 @@ function authResource($q, $http, umbRequestHelper, angularHelper) {
         'Could not send code');
     },
 
+    /**
+    * @ngdoc method
+    * @name umbraco.resources.authResource#get2FAProviders
+    * @methodOf umbraco.resources.authResource
+    *
+    * @description
+    * Verify the two-factor authentication code entered by the user against the provider
+    *
+    * ##usage
+    * <pre>
+    * authResource.verify2FACode(provider, code)
+    *    .then(function(data) {
+    *        //Do stuff ...
+    *    });
+    * </pre>
+    * @param {string} provider Name of the provider
```

Comment:
@poornimanayar I guess it should be "Name of the provider" here.

Reply:
Hi @bjarnef ,

I am not sure I understood what the comment is? 

Poornima

Reply:
You have written "provider Name of the provider". Shouldn't it just be "Name of the provider"?

Reply:
Thanks for clarifying @bjarnef . I am following the examples from other methods and it seems the convention for a parameter is `@param {type} name-of-parameter desription-of-parameter`

Reply:
@poornimanayar ahh yes, I think you're are right, that is how `ngdoc` handle the docs in the comments. Just looked wrong when viewing the changes on GitHub 🤦‍♂️

Reply:
No worries at all @bjarnef :)